Add Incident Management automation features (#4279)
mattermost/docs: source/administration/devops-command-center.rst

  * Optionally configure a broadcast channel to which status updates will be copied. If you are not a member of the configured broadcast channel, `Unknown Channel` will be shown instead of the channel name.
  * Optionally configure the default reminder timer used to prompt for regular updates. The reminder timer may be changed when a status update is written.
  * Optionally configure a template to use for the first status update. Subsequent status updates will start with the text of the most recent update.
8. Select the **Automation** tab.
9. Within the **Automation** tab, customize automatically triggered tasks:

>   * Optionally enable the **Invite members** toggle and select a set of members, which will be automatically invited to the incident channel when it starts

Reviewer: ```suggestion
  * Optionally enable the **Invite members** toggle and select a set of members. This set of members are automatically invited to the incident channel when the incident starts.
```

10. On the right-hand side of the screen, configure the permissions:

>   * Decide whether the automatically created incident channel should be public or private within the team.

Reviewer: ```suggestion
  * Decide whether the automatically created incident channel should be Public or Private within the team.
```
